javatests/arcs/android/e2e/testapp/res/layout/test_activity.xml

```diff
@@ -9,197 +9,201 @@
   ~ grant found at
   ~ http://polymer.github.io/PATENTS.txt
   -->
-
-<LinearLayout
+<ScrollView
```

Comment:
Are changes in this file necessary?

Reply:
Not really, but they were helpful. (Disable diffing on whitespace to see the lines I actually added)

This put the test app's UI inside a scroll view, so that you can still push the buttons when the debug text is long. I found that when I ran the collection test manually, the debug output was very long and pushed the buttons offscreen so I couldn't tap them anymore.